frontend/components/SmileFooter/components/SmileJoinFooter/style.js

```diff
@@ -0,0 +1,77 @@
+import { css } from 'glamor';
+import config from '../../../../config';
+
+const { colorConfig } = config;
+
+const container = css({
+  color: `${colorConfig.contentFontColor}`,
+  position: 'fixed',
+  height: '52px',
+  fontSize: '14px',
+  bottom: '0',
+  width: '100%',
+  justifyContent: 'center',
+  alignItems: 'center',
+  display: 'flex',
+  margin: 'auto',
+  backgroundColor: '#fff',
+  boxShadow: '0 - 3px 12px 0 rgba(0, 0, 0, .04)',
+  zIndex: '1',
+}).toString();
+
+const button = css({
+  marginBottom: '0',
+  fontFamily: 'inherit',
+  fontWeight: '400',
+  fontSize: '14px',
+  lineHeight: '20px',
+  textAlign: 'center',
+  verticalAlign: 'middle',
+  cursor: 'pointer',
+  backgroundImage: 'none',
+  whiteSpace: 'nowrap',
+  textOverflow: 'ellipsis',
+  overflow: 'hidden',
+  display: 'inline-block',
+  border: '1px solid transparent',
+  padding: '15px 32px',
+  transition: 'background-color .1s ease-in',
+  backgroundColor: `${colorConfig.buttonBackground}`,
+  textDecoration: 'none',
+  color: `${colorConfig.buttonFontColor}`,
+  outline: 'none',
+  borderRadius: '10px',
+  position: 'absolute',
+  top: '-62px',
+  marginLeft: 'auto',
+  marginRight: 'auto',
+  left: 0,
+  right: 0,
+  width: '66.66%',
+  boxShadow: '0 0 13px 0 rgba(0,0,0,.09)',
+  animation: 'fadeIn .2s ease-in-out',
+}).toString();
+
+const signIn = css({
+  fontWeight: '400',
+  fontSize: '14px',
+  lineHeight: '20px',
+  marginBottom: '8px !important',
+  display: '-webkit-inline-box',
```

Comment:
we should not use vendor prefixes. glamor is taking care of it
```suggestion
  display: 'inline-box',
```

Reply:
@Carsten89  I couldn't get the removal of -webkit- to work in my test environment.